prisma/seed.ts

```diff
@@ -28,11 +28,27 @@ async function main() {
     where: { slug: testFamily.slug },
   });
 
+  // eslint-disable-next-line fp/no-let
+  let family: Family;
   if (!isTestFamilyExists) {
-    await prisma.family.create({ data: testFamily });
+    family = await prisma.family.create({ data: testFamily });
     console.log(
       `test family created, test family name is "${testFamily.name}"`,
     );
+  } else {
+    family = isTestFamilyExists;
+  }
+
+  const isTestHouseholderExits = await prisma.householder.findFirst();
+
+  if (!isTestHouseholderExits) {
+    const testHouseholder = { name: faker.name.fullName() };
+    await prisma.householder.create({
+      data: { ...testHouseholder, family_id: family.id },
```

Comment:
it's better to add family_id to testHousholder and the directly pass it as data argument value. (right now it can be confusing)